schema_diagram.sql

```diff
@@ -0,0 +1,61 @@
+/* CREATE TABLES FOR DIAGRAM */
+
+-- patients
+CREATE TABLE patients (
+  id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(50) NOT NULL,
+  date_of_birth DATE NOT NULL,
+  PRIMARY KEY(id)
+);
+
+-- medical history
+CREATE TABLE medical_histories (
+  id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP NOT NULL,
+  patient_id INT NOT NULL,
+  status VARCHAR(250) NOT NULL,
+  FOREIGN KEY (patient_id) REFERENCES species(id);
+  PRIMARY KEY(id)
+);
+
+-- treatments
+CREATE TABLE treatments (
+  id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
+  type VARCHAR(250) NOT NULL,
+  name VARCHAR(250) NOT NULL,
+  PRIMARY KEY(id)
+);
+
+-- invoices
+CREATE TABLE invoices (
+  id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
+  total_amount FLOAT(2) NOT NULL,
+  generated_at TIMESTAMP NOT NULL,
+  payed_at TIMESTAMP NOT NULL,
+  medical_history_id INT NOT NULL,
+  PRIMARY KEY (id),
+  FOREIGN KEY (medical_history_id) REFERENCES medical_history(id)
```

Comment:
- Hey, please could you verify the table name `medical_history` ? I supposed you meant `medical_histories`

Reply:
Hi @zieeco thank you very much for your observations, I will make the adjustments for sure.

Reply:
@Favourezeugwa I'm glad that helped a little, you can reach out on slack if you need help.